app/views/TaxYearSummary.scala.html

```diff
@@ -209,8 +209,12 @@ <h2 class="govuk-heading-m">@messages(header)</h2>
                             (documentDetailWithDueDate.isLatePaymentInterest && !documentDetailWithDueDate.documentDetail.interestIsPaid))) {
                             <div class="govuk-tag govuk-tag--red">@messages("tax-year-summary.payments.overdue")</div>
                         }
-                        <div>
-                            <a class="govuk-link"  id="paymentTypeLink-@{index}" href="@getChargeSummaryHref(documentDetailWithDueDate)">@getChargeSummaryMessage(documentDetailWithDueDate)</a>
+                        <div id="paymentTypeText-@{index}">
+                            @if(documentDetailWithDueDate.documentDetail.originalAmount.getOrElse(throw MissingFieldException("Original Amount")) == 0) {
```

Comment:
I would suspect that we already have a story to refactor / remove code where we throwing exceptions in the view ( https://jira.tools.tax.service.gov.uk/browse/MISUV-6912 ), if not we need to create a story for this

Reply:
Created one. https://jira.tools.tax.service.gov.uk/browse/MISUV-7154